build/debs/BUILD

```diff
@@ -167,6 +174,12 @@ The Container Networking Interface tools for provisioning container networks.
     version_file = "//build:cni_package_version",
 )
 
+k8s_deb(
```

Comment:
So is there a deb rule but not an rpm rule? 

/cc @ixdy

Reply:
yes, that's correct.

It might help to know the k8s deb rule is [not very exciting](https://github.com/kubernetes/repo-infra/blob/master/defs/deb.bzl).

I expect there is no `k8s_rpm` because the metadata is different for `pkg_rpm` and is mostly found in the spec files so there's no need to wrap `pkg_rpm`. Might be useful for ensuring we only build for one architecture, but I don't know that it's worth the overhead.